test/sample/test_sample.py

```diff
@@ -96,6 +93,10 @@ def rosenbrock_problem():
 
 
 def create_petab_problem():
+    import petab
+
+    import pypesto.petab
```

Comment:
Do we want the imports now within the function? Would prefer global import if there is no strong argument against it.